lib/allure_report_publisher/lib/helpers/url_section_builder.rb

```diff
@@ -58,6 +58,13 @@ def comment_body(pr_comment = nil)
         url_section(job_entries: job_entries, separator: false)
       end
 
+      # Check if summary has failed tests
+      #
+      # @return [Boolean]
+      def summary_has_failures?
+        summary.status != "✅"
```

Comment:
I think it's better to check for failure explicitly via ❌. There is a middle status ❗which only indicates that report had flaky tests, we probably don't want to consider it as a failure